cincinnati/src/plugins/internal/channel_filter.rs

```diff
@@ -54,6 +57,9 @@ lazy_static! {
 #[async_trait]
 impl InternalPlugin for ChannelFilterPlugin {
     async fn run_internal(self: &Self, internal_io: InternalIO) -> Fallible<InternalIO> {
+        get_tracer()
+            .get_active_span()
+            .update_name("channel-filter".to_string());
```

Comment:
Use the predefined constant instead:
```suggestion
            .update_name(Self::PLUGIN_NAME.to_string());
```